fluentd/lib/parser_json_or_crio/lib/parser_json_or_crio.rb

```diff
@@ -0,0 +1,31 @@
+require 'fluent/parser'
+
+module Fluent
+  class ParserJsonOrCrio < Parser
+    Plugin.register_parser("json_or_crio", self)
+
+    config_param :crio_format, :string, default: '/^(?<time>.+) (?<stream>stdout|stderr)( (?<logtag>.))? (?<log>.*)$/'
+
+    def configure(conf={})
+      super
+      @crio_parser = Plugin.new_parser(@crio_format)
+      @crio_parser.configure(conf)
+      @json_parser = Plugin.new_parser('json')
+      @json_parser.configure(conf)
+    end
+
+    def parse(text)
+      if text[0] == '{'
```

Comment:
Do we need to check for nil or empty string? concerns with `text[0]`
```
2.4.0 :001 > t = ''
 => "" 
2.4.0 :002 > t[0]
 => nil 
```
Is it acceptible to assume its crio of the message is empty?

Reply:
The current parsers don't check for nil or empty string.
It is acceptable to assume crio if message is empty